query_server/sqllogicaltests/cases/ddl/alter_drop_after.slt

```diff
@@ -0,0 +1,28 @@
+#statement ok 
+#drop tenant if exists tenant001;
```

Comment:
Why comment it out here?

Reply:
because the tenant with the field "drop_after" cannot delete immediately. if you delete it at first, you can not create it then.

Reply:
Add "drop tenant tenant001;" at the end of the use case to ensure that the use case can be repeated